src/hotspot/share/runtime/synchronizer.hpp

```diff
@@ -123,7 +123,8 @@ class ObjectSynchronizer : AllStatic {
   static void notifyall(Handle obj, TRAPS);
 
   static bool quick_notify(oopDesc* obj, JavaThread* current, bool All);
-  static inline bool quick_enter(oop obj, JavaThread* current, BasicLock* Lock);
+
```

Comment:
Why add the extra blank line?

Reply:
The `quick` grouping seemed inconsistent. Was probably thinking about moving down `quick_enter_legacy`. But in the end, there is no obvious grouping. The newline must have slipped through, I'll revert it. 
```suggestion
```